app/Resources/views/base.html.twig

```diff
@@ -154,6 +154,12 @@
             <script>
                 $(document).ready(function() {
                     hljs.initHighlightingOnLoad();
+
+                    $('[data-toggle="confirmation"][data-message]').on('click', function (event) {
```

Comment:
this should not be done on click. It should be done on form submission. Otherwise other ways to submit the form will not trigger the confirmation